doc/api/buffer.markdown

```diff
@@ -541,6 +541,47 @@ console.log(bufA.length);
 // 42
 ```
 
+### Class Method: Buffer.encode(str[, str_encoding][, target_encoding])
+
+* `str` {String} A source string
+* `str_encoding` {String} The character encoding for `str`. Default = `'utf8'`
+* `target_encoding` {String} The target character encoding. Default = `'utf8'`
+
+Historically, a common use case for `Buffer` instances has been to reinterpret
```

Comment:
Suggest we don't use this kind of time-based language in the docs because it'll date pretty quickly. Perhaps instead just refer to the _alternative_ way to achieve this with `Buffer.from().toString()`. Even now the "it is not uncommon to find ..." bit is not strictly accurate because `Buffer.from()` is so new that it _is_ uncommon to find it in actual code.